backend/src/v2/driver/driver.go

```diff
@@ -512,6 +512,39 @@ func extendPodSpecPatch(
 		}
 	}
 
+	// Get config map mount information
```

Comment:
nit: maybe we can refactor these two for loops into functions so that we can also reuse them for secret mount and secret env